glide.yaml

```diff
@@ -72,3 +72,14 @@ import:
   subpackages:
   - compute/metadata
   - internal
+- package: k8s.io/kubernetes
+  version: e4a1ca7105fa9a3075fbf13a3286f8dddd59a658
```

Comment:
Side note: it took way to long to update vendor (multiple hours). I have no idea what I'm doing with glide. Any straightforward doc on updating deps I should look at?